homeassistant/components/frontend/__init__.py

```diff
@@ -631,7 +630,8 @@ def add_prefix(self, prefix: str) -> None:
 
     def get_info(self) -> dict[str, list[str]]:  # type: ignore[override]
         """Return a dict with additional info useful for introspection."""
-        return {"panels": list(self.hass.data[DATA_PANELS])}
+        panels = self.hass.data[DATA_PANELS]
+        return {"panels": list(panels)}
```

Comment:
Unfortunately necessary to split as `HassKey` is invariant and `list(...)` expects an `Iterable`. The alternative would be to add a type ignore.